### ISSUE TITLE
Some small improvement requests from the community for the referenda tab

### ISSUE THREAD

**Issue by rrtti** [polkadot-js/apps#8419]
Opening this issue after reading some requests from the community for information on Polkadot.js Apps, particularly on Gov2 referenda tab. 

1. Next to the track name, some description of the track might be useful - if it helps, short descriptions can be found on [this spreadsheet](https://docs.google.com/spreadsheets/d/1V5o-VYVT2C0B1zj_9VnWjti6USwig1v1ND-96otA1Ug/edit?usp=sharing), column A. 

<img width="224" alt="Screenshot 2022-11-25 at 10 49 30" src="https://user-images.githubusercontent.com/22815262/203952759-09d04c6c-d63c-4941-857a-baa5ee417483.png">

2. It is hard for some users to understand the meaning of the proposer accounts in a referendum: 

<img width="293" alt="Screenshot 2022-11-25 at 10 50 53" src="https://user-images.githubusercontent.com/22815262/203953121-70feca2d-6b1f-447a-8930-c5523dd0d744.png">

We could add some info, stating these two accounts are the proposer and the account that paid the decision deposit for the proposal to pass to deciding phase. 

On the "Submit Proposal" window: 

3. Is it possible to compose the preimage in the window as well? or is it a requisite to compose in the preimage tab and have it submitted before submitting the proposal? Is there a way to allow a proposer to compose the preimage in the window itself?

<img width="612" alt="Screenshot 2022-11-25 at 10 54 26" src="https://user-images.githubusercontent.com/22815262/203954071-875242f0-3296-4778-8c4a-354586712713.png">

4. When choosing the Submission Track in the Submit Proposal window, it would be helpful to have some sort of short information for each track, if space allows: 

<img width="585" alt="Screenshot 2022-11-25 at 10 55 50" src="https://user-images.githubusercontent.com/22815262/203954348-644fc7d3-2dcc-44ba-bfac-03798de3682b.png">

5. Can we add a note on the enactment block number entry  (that is predetermined in advance) for users to note this might change with different tracks, if thats the case? 

<img width="577" alt="Screenshot 2022-11-25 at 10 57 18" src="https://user-images.githubusercontent.com/22815262/203954669-9180cf36-a888-4aed-9994-4f50b4c1b62d.png">

**Comment by jacogr** [comment]
Just one comment on sheets -

Unless it is in the actual on-chain metadata, we should really not display it. Managing these between all the chains in existence is an absolute nightmare. Each is different, each has different requirements/descriptions/parameters. So it is very unlikely that it will be copied from a spreadsheet. The code (mapped to specific versions of the runtime) is the _only_ source of truth if not supplied in the actual metadata.

So... we already have forced copies of [track -> origin](https://github.com/polkadot-js/apps/blob/1d032ac66576414852bca947310b6437aee592e0/packages/apps-config/src/api/params/tracks.ts#L10-L40) which is giving sleepless nights since it it all about manually tracking all PRs and applying changes as they may go in. This is completely non-scalable.

Having said that, the above link _could_ be extended to have descriptions alongside the origins, but once again, along with the origins above it is bound to fail multiple times in the chain lifetime, especially if things ever change between runtime versions.

TL;DR I'm anxiously waiting for the above approach to fail completely since I missed some PR along the way... (Or non-backwards compatible changes were made between runtimes, it assumes all the same, forever...)
